etc/authordb.yaml

```
@@ -232,6 +232,7 @@ affiliations:
   Santiago-Millenium: Millennium Institute of Astrophysics, Nuncio Monse{\~{n}}or S{\'{o}}tero
     Sanz 100, Of 104, Providencia, Santiago, Chile
   SAO: Smithsonian Astrophysical Observatory, 60 Garden St., Cambridge MA 02138, USA
+  SBU: Department of Physics and Astronomy, Stony Brook University, Stony Brook, NY 11794, USA
```

```suggestion
  StonyBrookU: Department of Physics and Astronomy, Stony Brook University, Stony Brook, NY 11794, USA
```
how about this for the key in the affiliation to make it clearer for people that might not know what SBU is?